applications/MetisApplication/tests/test_MetisApplication.py

```diff
@@ -7,7 +7,6 @@
 
 # Import the tests o test_classes to create the suits
 from generalTests import KratosMetisGeneralTests
```

Comment:
could you delete this file as it does nothing?

also `test_MetisApplication.py` is unused so can be removed too I would suggest

Reply:
done